src/ansys/fluent/core/solver/flobject.py

```diff
@@ -338,6 +353,29 @@ class Real(SettingsBase[RealType], Numerical):
     expression values.
     """
 
+    def get_state(self) -> StateT:
```

Comment:
@ansaboutin As discussed, let's rename this to `get_state_as_quantity` so that the current `get_state` behaviour is preserved. Thereby, we can also eliminate the `real_with_units` stuff.